README.md

```diff
@@ -19,7 +19,7 @@ Code manager: Jeff Ator
 <pre>
 git clone https://github.com/noaa-emc/nceplibs-bufr
 mkdir build && cd build
-cmake -DCMAKE_INSTALL_PREFIX=./install -DENABLE_TESTS=ON nceplibs-bufr
+cmake -DCMAKE_INSTALL_PREFIX=./install -DENABLE_TESTS=ON ../nceplibs-bufr
```

Comment:
I think it should just be ".."

Reply:
If you just use "..", then how does cmake know to drill down into the nceplibs-bufr subdirectory?  I think you still need the full "../nceplibs-bufr", and I can confirm that it does work that way, so my vote would be to leave it there.